docs/internal-documentation/maintenance/ReleaseProcess.md

```diff
@@ -123,13 +143,13 @@ zonemaster-gui:
     git tag v1.0.0
 	git push origin --tags
 
```

Comment:
Insert below:

"TBD! Write a description how to set release in Github to get a nice presentation."